src/Analysis/Engine/Impl/Values/BuiltinClassInfo.cs

```diff
@@ -158,7 +159,22 @@ public BuiltinClassInfo(IPythonType classObj, PythonAnalyzer projectState)
         }
 
         public override IAnalysisSet GetIndex(Node node, AnalysisUnit unit, IAnalysisSet index) {
-            // TODO: Needs to actually do indexing on type
+            try {
+                var indices = index.Select(x => x.GetConstantValue()).ExcludeDefault().OfType<int>().ToArray();
+                if (this is IPythonSequenceType seqt) {
```

Comment:
Is it possible? What type is derived from `BuiltinClassInfo` and implements `IPythonSequenceType`?

Reply:
Looks like no one